api/src/main/resources/leonardo.yaml

```diff
@@ -1681,7 +1689,8 @@ components:
           $ref: "#/components/schemas/AuditInfo"
         runtimeConfig:
           type: object
-          description: oneOf GceConfig, GceWithPdConfig, MachineConfig
+          description: oneOf GceWithPdConfigInResponse, MachineConfig
+          # Leo's version -> $ref: "#/components/schemas/OneOfRuntimeConfigInResponse"
```

Comment:
rearranging our code to use this structure will be necessary to use the client.  In this case, I think it will be simpler to wait.